site/getting-started/setup.md

```diff
@@ -95,6 +95,10 @@ To read or write any data to a blockchain, you will need an Alchemy API Key and
 
 ## 3. Query your Smart Account Address
 
+::: tip
+Don't use the example below with a hardcoded private key in production. Follow the [Alchemy Signer guide](/signers/alchemy-signer) to build a secure user friendly Embedded Account!
```

Comment:
Don't use hardcoded private keys in production, like the example below. Instead, follow this [Alchemy Signer ...